src/main/webapp/WEB-INF/jsp/account/account_form.jsp

```diff
@@ -7,7 +7,7 @@
 <%@taglib prefix="asset" uri = "/WEB-INF/asset.tld"%>
 
 <!DOCTYPE html>
-<html>
+<html lang="${pageContext.response.locale}">
```

Comment:
`lang` describes the language of the content iirc. (used by chrome translate popup for example)

I would prefer a hard set of locales (currently `de` and `en`) and a fallback to the default locale when the user locale is not in the given set.

Reply:
can you give me a example?

Reply:
something like

```java
public class SupportedLanguagesInterceptor implements HandlerInterceptor {

    @Override
    public void postHandle(HttpServletRequest request, HttpServletResponse response, Object handler,
                           ModelAndView modelAndView) {

        if (modelAndView != null) {

			// available translations in our app
			// -> do it dynamically instead of hard coded
            modelAndView.addObject("languages", Arrays.asList("de", "en"));

			// language selected by the user (default: browser locale)
            modelAndView.addObject("currentLanguage", "de");
        }
    }
}
```

```html
<!DOCTYPE html>
<html lang="${currentLanguage}">
```

Reply:
Hm. Do you know that <html lang="${pageContext.response.locale}"> is not hard coded? It depends on the defined browser language ? oO Maybe we should talk in person :)

Reply:
> It depends on the defined browser language

yep.

however we only have translations for german and english right now. not for french, austria, polish, $browserLocale

the `lang` attribute describes the language of the rendered content.

Reply:
ok I got it :) It was too late to see that :)